src/ucp/api/ucp.h

```diff
@@ -213,6 +213,24 @@ enum ucp_ep_params_flags_field {
 
 
 /**
+ * @ingroup UCP_ENDPOINT
+ * @brief @anchor ucp_ep_close_nb_mode Close UCP endpoint modes.
+ *
+ * The enumeration allows specifying behavior of @ref ucp_ep_close_nb routine.
+ */
+enum {
+    UCP_EP_CLOSE_MODE_FORCE         = 0, /**< @ref ucp_ep_close_nb releases
+                                              the endpoint without any
+                                              confirmation from the peer. All
+                                              outstanding requests will be
+                                              completed with
+                                              @ref UCS_ERR_CANCELED error. */
+    UCP_EP_CLOSE_MODE_FLUSH         = 1  /**< @ref ucp_ep_close_nb flushes all
+                                              outstanding operations. */
```

Comment:
Suggest:
 ...schedules flushes on all outstanding operations.

I assume that is the semantic for MODE_FLUSH.  If so, then I think this will make it clearer.